Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue5555.cs

```diff
@@ -79,6 +81,7 @@ protected override void Init ()
 							GC.Collect();
 							GC.WaitForPendingFinalizers();
 							GC.Collect();
+							GC.WaitForPendingFinalizers();
```

Comment:
And I'm not sure about the point of this line. Why wait for the finalizers again? Never seen it this way before.